Dx fixing direct credit card feedback (#311)
SwedbankPay/developer.swedbankpay.com: payments/card/direct.md

        "acquirerTransactionTime": "2020-03-10T14:13:52Z",
        "nonPaymentToken": "ed4683a8-6d2a-4a14-b065-746a41316b8f",
        "transactionInitiator": "CARDHOLDER",
        "id": "/psp/creditcard/payments/{{ page.payment_id }}/authorizations/{{ page.transaction_id }},
        "transaction": {
            "id": "/psp/creditcard/payments/{{ page.payment_id }}/transactions/{{ page.transaction_id }}",
            "created": "2020-03-10T13:13:52.2767764Z",
            "updated": "2020-03-10T13:13:53.280398Z",
            "type": "Authorization",
            "state": "Completed",
            "number": 70100366754,
            "amount": 4201,
            "vatAmount": 0,
            "description": "Test transaction",
            "payeeReference": "1583846025",
            "isOperational": false,
            "operations": [
                {
                    "method": "PATCH",

>                     "href": "https://api.stage.payex.com/psp/creditcard/payments/{{ page.payment_id }}/authorizations/84c440a1-5745-4a88-f4ed-08d7c4dcbf36",

Reviewer: ```suggestion
                    "href": "https://api.stage.payex.com/psp/creditcard/payments/{{ page.payment_id }}/authorizations/{{ page.transaction_id }}",
```

}
```

## Step 2b: Create an authorization transaction with 3-D secure

> If 3-D secure authentication is needed, the `rel` is set to

Reviewer: ```suggestion
If 3-D Secure authentication is needed, the `rel` is set to
```

`redirect-authentication` and the `state` is `AwaitingActivity`. This means that

> you will be redirected to complete the 3-D secure authentication. See the

Reviewer: ```suggestion
the payer will have to be redirected to complete the 3-D secure authentication. See the
```

        "panToken": "cca2d98d-8bb3-4bd6-9cf3-365acbbaff96",
        "panEnrolled": true,
        "acquirerTransactionTime": "0001-01-01T00:00:00Z",

>         "id": ""/psp/creditcard/payments/{{ page.payment_id }}/{{ page.transaction_id }}",

Reviewer: ```suggestion
        "id": ""/psp/creditcard/payments/{{ page.payment_id }}/authorizations/{{ page.transaction_id }}",
```

        "transaction": {

>             "id": ""/psp/creditcard/payments/{{ page.payment_id }}/{{ page.transaction_id }}",

Reviewer: ```suggestion
            "id": ""/psp/creditcard/payments/{{ page.payment_id }}/transactions/{{ page.transaction_id }}",
```

            "created": "2020-03-10T13:15:01.9586254Z",
            "updated": "2020-03-10T13:15:02.0493818Z",
            "type": "Authorization",
            "state": "AwaitingActivity",
            "number": 70100366758,
            "amount": 4201,
            "vatAmount": 0,
            "description": "Test transaction",
            "payeeReference": "1583846100",
            "isOperational": true,
            "operations": [
                {
                    "method": "GET",

>                     "href": "https://api.stage.payex.com/psp/creditcard/confined/payments/authorizations/authenticate/{{ page.payment_id }}",

Reviewer: ```suggestion
                    "href": "https://api.stage.payex.com/psp/creditcard/confined/payments/authorizations/authenticate/{{ page.transaction_id }}",
```
